.github/workflows/cml.yaml

```diff
@@ -1,52 +1,35 @@
 name: train-my-model
-
 on: [push]
-
 jobs:
   run:
     runs-on: [ubuntu-latest]
-  
     steps:
       - uses: actions/checkout@v2
-
       - uses: iterative/setup-cml@v1
-
-      - name: cml_run
-        env:
-          repo_token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Install
         run: |
+          cml ci  # complete repository setup for CML
           pip install -r requirements.txt
-
-          # Reproduce pipeline if any changes detected in dependencies
-          dvc repro
-
+      - name: Reproduce pipeline if any changes detected in dependencies
+        run: dvc repro
+      - name: CML report
+        env:
+          REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
           # Publish confusion matrix diff
-          echo -e "## Plots\n### Class confusions" >> report.md
+          echo "## Plots" > report.md
+          echo "### Class confusions" >> report.md
           dvc plots diff \
             --target output/predictions.json \
             --show-vega > vega.json
-          vl2png vega.json -s 1.5 | cml-publish --md >> report.md
-          # Add figure to report
-          cat output/metrics.json >> report.md
-          cml-send-comment report.md
-
-      - name: Commit files
-        id: commit
+          vl2png vega.json -s 1.5 | cml publish --md >> report.md
+          echo "## Metrics" >> report.md
+          dvc metrics show --md >> report.md
```

Comment:
I assume `dvc metrics show --md >> report.md` works? Rather than the previous `cat output/metrics.json >> report.md`

Reply:
Probably? Assumptions are dangerous in CI/CD workflows; empyrical testing will dissipate our doubts.